CMakeLists.txt

```diff
@@ -195,12 +192,17 @@ install(TARGETS dronecore
 
 install(FILES
     include/dronecore.h
-    include/device.h
-    ${CMAKE_CURRENT_BINARY_DIR}/include/device_plugin_container.h
+    core/device.h
     ${plugin_header_paths}
     DESTINATION "include/dronecore"
 )
 
+set(dronecore_install_include_dir "include/dronecore")
+set(dronecore_install_lib_dir ${lib_path})
+
+add_subdirectory(plugins/action)
```

Comment:
So this allows you to compile action separately, which is good. But has to already have been compiled before you build the core right?
So in other words if a plugin wasn't known at compile time for the core you can't add it later? WOuldn't it be better for the core not to know about dependency plugins?